oozie/oozie.sh

```diff
@@ -125,7 +125,29 @@ function install_oozie() {
     --configuration_file "/etc/hadoop/conf/core-site.xml" \
     --name 'hadoop.proxyuser.oozie.groups' --value '*' \
     --clobber
+    
+  bdconfig set_property \
+    --configuration_file "/etc/oozie/conf/oozie-site.xml" \
+    --name 'oozie.service.HadoopAccessorService.supported.filesystems' --value "hdfs,gs" \
+    --clobber
 
+  bdconfig set_property \
+    --configuration_file "/etc/hadoop/conf/core-site.xml" \
+    --name 'fs.AbstractFileSystem.gs.impl' \
+    --value 'com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS' \
+    --clobber
+  
+  local gcs_connector_dir="/usr/local/share/google/dataproc/lib"
+  if [[ ! -d \$install_dir ]]; then
+    gcs_connector_dir="/usr/lib/hadoop/lib"
+  fi
+  
+  local gcs_connector_path="$gcs_connector_dir/gcs-connector.jar"
+  if [ -e $gcs_connector_path ]
+  then
+    cp $gcs_connector_path /usr/lib/oozie/lib/gcs-connector.jar
+  fi
```

Comment:
I think it's better to remove this check and just try to copy GCS connector - if it fails it means that configuration/script is wrong and we need to fix it.

Reply:
done